### PR TITLE
Updated for new Docker image

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated for new Docker image

### CODE REVIEW

File: docs/Getting-Started/Run-Docker-Image.md [hunk 1]
```diff
@@ -32,63 +31,53 @@ docker run pegasyseng/pantheon:{{ versions.pantheon_stable }}
...
+    ```
+     
 ## Starting Pantheon 
 
 ### Run a Node for Testing 
 
 To run a node that mines blocks at a rate suitable for testing purposes with WebSockets enabled: 
 ```bash
-docker run -p 8546:8546 --mount type=bind,source=/<myvolume/pantheon/testnode>,target=/var/lib/pantheon pegasyseng/pantheon:latest --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-http-cors-origins="all" --rpc-ws-enabled --network=dev
+docker run -p 8546:8546 pegasyseng/pantheon:latest --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --data-path=/mynode/data --rpc-ws-enabled --network=dev
```
Comment (ajsutton) [inline docs/Getting-Started/Run-Docker-Image.md]: You'd still want to bind a volume so the data isn't lost when Docker shuts down.  So:
```suggestion
docker run -p 8546:8546 --mount type=bind,source=/<myvolume/pantheon/testnode>,target=/mynode/data pegasyseng/pantheon:latest --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --data-path=/mynode/data --rpc-ws-enabled --network=dev
```

Or you could use the new default data-path:
```
docker run -p 8546:8546 --mount type=bind,source=/<myvolume/pantheon/testnode>,target=/opt/pantheon/database pegasyseng/pantheon:latest --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-ws-enabled --network=dev
```

File: docs/Getting-Started/Run-Docker-Image.md [hunk 2]
```diff
@@ -32,63 +31,53 @@ docker run pegasyseng/pantheon:{{ versions.pantheon_stable }}
...
 ```
 
 ### Run a Node on Rinkeby Testnet 
 
 To run a node on Rinkeby: 
 ```bash
-docker run -p 30303:30303 --mount type=bind,source=/<myvolume/pantheon/rinkeby>,target=/var/lib/pantheon pegasyseng/pantheon:latest --network=rinkeby
+docker run -p 30303:30303 pegasyseng/pantheon:latest --network=rinkeby --data-path=/mynode/data
```
Comment (ajsutton) [inline docs/Getting-Started/Run-Docker-Image.md]:
```suggestion
docker run -p 30303:30303 --mount type=bind,source=/<myvolume/pantheon/rinkeby>,target=/opt/pantheon/database pegasyseng/pantheon:latest --network=rinkeby
```

File: docs/Getting-Started/Run-Docker-Image.md [hunk 3]
```diff
@@ -32,63 +31,53 @@ docker run pegasyseng/pantheon:{{ versions.pantheon_stable }}
...
-
+To run a node on Ethereum mainnet with the HTTP JSON-RPC service enabled: 
 ```bash
-docker run -p 30303:30303 --mount type=bind,source=/<myvolume/pantheon>,target=/var/lib/pantheon pegasyseng/pantheon:latest
-```
-
-To run a node on mainnet with the HTTP JSON-RPC service enabled: 
-```bash
-docker run -p 8545:8545 -p 30303:30303 --mount type=bind,source=/<myvolume/pantheon>,target=/var/lib/pantheon pegasyseng/pantheon:latest --rpc-http-enabled
+docker run -p 8545:8545 -p 30303:30303 pegasyseng/pantheon:latest --rpc-http-enabled --data-path=/mynode/data
```
Comment (ajsutton) [inline docs/Getting-Started/Run-Docker-Image.md]:
```suggestion
docker run -p 8545:8545 --mount type=bind,source=/<myvolume/pantheon/rinkeby>,target=/opt/pantheon/database  -p 30303:30303 pegasyseng/pantheon:latest --rpc-http-enabled
```

File: docs/Getting-Started/Run-Docker-Image.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Getting-Started/Run-Docker-Image.md]: Good point.  Have updated.

**Pull request merged**